src/collections/members/Aboobaker-Siddiq-R/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Aboobaker Siddiq R
+position: Contributor
+image_path: ./Aboobaker-Siddiq-R.webp
+github: aboobakersiddiqr63
+linkedin: aboobakersiddiqr
+layer5: 846d0933-57da-44ff-9ae8-098007e03cb3
+location: Chennai, India
+bio:  I'm Aboobaker Siddiq, a cloud engineer and full stack developer. My passion for open source technologies drives me to craft innovative solutions that thrive in the digital landscape. I'm dedicated to creating robust software and web applications that seamlessly blend cutting-edge functionality with exceptional user experiences.
+badges: 
+  - meshery
```

Comment:
@Harsha-10 can you also add here doc badge. Aboobaker is consistent contributor there also